packages/react-composites/src/composites/CallComposite/styles/LocalPreview.styles.ts

```diff
@@ -1,19 +1,32 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import { mergeStyles } from '@fluentui/react';
+import { IStyle, mergeStyles } from '@fluentui/react';
+
+const localPreviewContainerStyle: IStyle = {
+  borderRadius: '.25rem',
+  minWidth: '12.5rem'
+};
 
 /**
  * @private
  */
-export const localPreviewContainerStyle = mergeStyles({
-  borderRadius: '.25rem',
+export const localPreviewContainerStyleDesktop = mergeStyles({
+  ...localPreviewContainerStyle,
   width: '25rem',
-  minWidth: '12.5rem',
   height: '18.75rem',
   padding: '0.5rem'
 });
 
+/**
+ * @private
+ */
+export const localPreviewContainerStyleMobile = mergeStyles({
+  ...localPreviewContainerStyle,
+  width: '100%',
```

Comment:
For my better understanding: Could you explain why we fill all the space on mobile, but have a fixed size on desktop? And if the explanation is at all non-trivial, consider adding it as comment in the code.

Reply:
Desktop viewports could be huge (so local preview could be enormous), mobile viewports are small so we want to fill the space